lib/active_merchant/billing/credit_card.rb

```diff
@@ -245,7 +245,7 @@ def last_name?
       #
       # @return [String] the full name of the card holder
       def name
-        [first_name, last_name].compact.join(' ')
+        [first_name, last_name].compact.reject{ |n| n == "" }.join(' ')
```

Comment:
Am I missing an edge case that negates the clarity of logic like this (vs. existing array-based approach)?

```ruby
"#{first_name} #{last_name}".strip
```